libraries/AP_InertialSensor/AP_InertialSensor_Invensense.cpp

```diff
@@ -658,6 +662,7 @@ void AP_InertialSensor_Invensense::_read_fifo()
     bool need_reset = false;
 
     if (!_block_read(MPUREG_FIFO_COUNTH, rx, 2)) {
+        _seqcnt = 0;
```

Comment:
I thought this indicated a possible discontinuity in sampling; haven't seen it happen yet.